README.md

```diff
@@ -671,32 +743,38 @@ only marking here that you are likely to hear both.
     Say you take three months of parental leave in your first year.
     Does your cliff extend to 15 months?
 
-- 🄳 Vesting *can* be triggered outside of the vesting schedule.
-  [**Acceleration**](http://www.danshapiro.com/blog/2012/04/vesting-is-a-hack/) dictates that
-  vesting will be triggered if a company is sold (**single trigger**) or if it’s sold and
-  you’re fired (**double trigger**).
+- 🄳 Vesting may be triggered outside of the vesting schedule, depending on the terms of an
+  agreement. [Accelerated vesting](http://www.danshapiro.com/blog/2012/04/vesting-is-a-hack/)
```

Comment:
And depending on the conditions in which you leave the company. I have notes on this and will add here!